satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -730,3 +730,38 @@ read count (
 )
 
 create user_credit ()
+
+//--- bucket metadata table ---//
+
+model bucket_metadata (
+	key project_id name
+
+	field project_id		blob
+	field name  			blob
+
+	field segment_size 		int
```

Comment:
can we name this preferred or default segment size? just to indicate that no matter what this value is, not all segments are necessarily that size?